plugins/explain.py

```diff
@@ -0,0 +1,87 @@
+import re
+
+from errbot import BotPlugin, re_botcmd
+
+
+class Explain(BotPlugin):
+    """
+    Explain various terms
+    """
+
+    MSGS = {
+        'review': 'After creating your `Pull Request`, it is under the review '
+                  'process. This can be deduced from the `process/pending '
+                  'review` label. Now you have to wait for the reviewers to '
+                  'review your PR. You should *not* ask for reviews on our '
+                  'Gitter channel - we review those PRs continuously.\n\n'
+                  "We're usually swamped with reviews, while you are waiting "
+                  "**please review other people's PRs** at [coala.io/review]"
+                  '(https://coala.io/review): that helps you and will make '
+                  'your review happen faster as well. As a rule of thumb, '
+                  '*for every review you receive, give at least one review '
+                  'to someone else!*\n\nFor a good review, look at every '
+                  'commit on its own and place `<sha> is ready` or `<sha> '
```

Comment:
hmm.. no idea why this, I copied all of the contents/messages from old cobot explain command